docs/walk-through.md

```diff
@@ -257,15 +275,39 @@ $ curl \
     --silent        `# Don't write progress bar to stderr` \
     --header "Content-Type: application/json; odata=minimalmetadata" \
     --data "{'applicationId':'$applicationId', 'token':'$AZ_BATCH_SOFTWARE_ENTITLEMENT_TOKEN'}" \
-    $AZ_BATCH_ACCOUNT_URL/softwareEntitlements?api-version=$apiVersion \
+    $AZ_BATCH_ACCOUNT_URL/softwareEntitlements?api-version=2017-09-01.6.0 \
     | jq
+```
+
+The response body should be similar to:
+```json
+{
+  "id": "entitlement-695c822b-470c-4fd1-b34d-9b21435f57b2", // every entitlement has a unique identifier
+  "expiry": "2019-01-16T01:06:07+00:00"                     // reflects the expiry of the token, so all requests passing the same token will have the same expiry
```

Comment:
We should probably remove this in a subsequent PR. Returning the `expiry` value is part of the pre-leasing "V2" API which was never deployed in production.